scripts/launch-xrd

```diff
@@ -250,18 +241,15 @@ while [[ $# -gt 0 ]]; do
            shift
            ;;
         *)
-            if [[ ! $IMAGE ]]; then
-                IMAGE=$1
-                shift
-            else
-                echo "Unexpected argument '$1'" >&2
-                bad_usage
-            fi
-            ;;
+           CTR_MGR_ARGS+=("$1")
+           shift
+           ;;
     esac
 done
 
-if [[ ! $IMAGE ]]; then
+if [[ ! $IMAGE && -n ${CTR_MGR_ARGS[-1]} ]]; then
+    IMAGE="${CTR_MGR_ARGS[-1]}"
```

Comment:
There is a slight issue here where passing `launch-xrd xrd --unknown` won't get to `docker run ... xrd --unknown` because the word `--unknown` will be guessed to be the image and in the case where the platform is not know, the following code will fail
```bash
# Obtain the platform from the chosen image.
if [[ $PLATFORM == "" ]]; then
    if ! PLATFORM="$(_get_platform "$IMAGE")"; then
        bad_usage
    fi
fi
```
Either we acknowledge this doesn't work or we choose the last unknown argument that doesn't begin `--` as the image. Potentially worth checking with legaul.

Reply:
Or we can just update the error output of `_get_platform` (before `bad_usage`) to say something like "Cannot get platform of $IMAGE, please specify --platform"

Reply:
But then wouldn't the problem be that if users did `--unknown /foo/bar.txt`, then the `/foo/bar.txt` would be the IMG. Users would have to do `--unknown=/foo/bar.txt`

Reply:
Yes, that would be an issue too. My suggestion would just be best effort, so in the end we could still fail earlier than we would like.

Considering the `_get_platform` error output recommends passing `--platform` manually if it fails this may not be too much of a problem as the user will just be prompted to specify the platform the manual way.